0000-unified-machine-word-trait.md

```diff
@@ -0,0 +1,196 @@
+- Feature Name: unified_machine_word_arithmetic
+- Start Date: 2016-07-13
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Unify functionality peculiar to `i8`…`i64` and `u8`…`u64` in a trait containing the
+family of `overflowing`/`checked`/`wrapping`/`saturating` variants of arithmetic operations,
```

Comment:
I don't think there are existing traits for the overflowing/checked/wrapping/saturating methods.